t/lib/t/MusicBrainz/Server/Data/Instrument.pm

```diff
@@ -158,4 +158,6 @@ test 'Merge instruments' => sub {
             },
         ],
     );
-}
+};
+
+1;
```

Comment:
A couple files in this commit don't end in a newline, it seems (t/script/SubscriptionEmails.t too).